### DISCUSSION TITLE
Argument Name Mismatch

### DISCUSSION THREAD

**Discussion by ghley** [DanielChappuis/reactphysics3d#216] [Q&A] [answered]
Hi there,

In CollisionShape.h the method:

    virtual bool testPointInside(const Vector3& worldPoint, Collider* collider) const=0;

specifically calls for a "worldPoint" but in BoxShape.h it's renamed to localPoint:

    inline bool BoxShape::testPointInside(const Vector3& localPoint, Collider* collider) const {
        return (localPoint.x < mHalfExtents[0] && localPoint.x > -mHalfExtents[0] &&
            localPoint.y < mHalfExtents[1] && localPoint.y > -mHalfExtents[1] &&
            localPoint.z < mHalfExtents[2] && localPoint.z > -mHalfExtents[2]);
    }

Given the code and assuming that the physics engine works, I assume the naming in CollisionShape.h is wrong?

**Comment by DanielChappuis** [comment, marked as answer]
Hello,

Thanks for reporting that. 

You are right, the parameter in the CollisionShape.h file should be **localPoint** instead of **worldPoint**.
I have commited a [fix](https://github.com/DanielChappuis/reactphysics3d/commit/ce0bb3b9a660c2a13dc077a786dd41692ec92eff) for that. This change will be available in the next release of the library.
